src/codegen/__main__.py

```diff
@@ -0,0 +1,48 @@
+import logging
```

Comment:
FYI: We do not want to include a `generated`, `codegen`, or `src` directory inside the `nidaqmx` package that we upload to PyPI. 

I ran `poetry build` and opened the resulting wheel with 7-Zip, and I don't see anything out of place.

Then I tried deleting the top-level `nidaqmx/`, moving `generated/*` to `generated/nidaqmx/*`, adding this to `pyproject.toml`, and running `poetry build`:
```
packages = [
    { include = "nidaqmx", from = "generated" }
]
```

The resulting wheel still has everything in the right place, so I think this directory structure will work once you move `generated/*` to `generated/nidaqmx/*`.